### PR TITLE
feat(STN-306): background color for Row Paragraph type

### PR DESCRIPTION
**Pull request by silverli**
# READY FOR REVIEW

## Summary
Adds a new field for the paragraph style on the Row paragraph type.

**Browser testing**
- [x] Chrome
- [x]  Firefox
- [x] Safari
- [x] IE11

## Steps to Test
1. In humsci_basic, run `npm run test`
2. If you have not synced the latest from econ, I would recommend doing that now, on my branch: `lando blt drupal:sync --site=economics --sync-files --partial`.
3. If you have pulled from econ recently, then you can just import the new database changes: `lando drush @economics.local config-import --partial`
4. After logging into the site, go to a basic page, and either edit or add a new Paragraph "Row" type. Make sure to check for the existence of the new select field called "Paragraph Style". Select a paragraph style and then save. Check your page to see if the changes were applied.
5. Check both limited width and full width options. Also make sure to check a row on a page with a sidebar (both options should become limited width).

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: config/default/field.field.paragraph.hs_row.field_paragraph_style.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+label: 'Paragraph Style'
+description: 'Allows you to add styles to the background of the row.'
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  allowed_values:
+    -
+      value: well
```
Comment (heatherdesigns) [inline config/default/field.field.paragraph.hs_row.field_paragraph_style.yml:23]: I assume this is similar to the `hb-well` utility class.

File: docroot/modules/humsci/hs_paragraph_types/modules/hs_row/modules/hs_row_display/hs_row_display.module [hunk 2]
```diff
@@ -8,6 +8,20 @@
 use Drupal\Core\Entity\EntityInterface;
 use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
 
+/**
+ * Implements template_preprocess_paragraph()
```
Comment (heatherdesigns) [inline docroot/modules/humsci/hs_paragraph_types/modules/hs_row/modules/hs_row_display/hs_row_display.module]: I think recall something about comments needing to end with. `.`.  Maybe? 🤔 See lines 26 and 35. 
Maybe it was in a theme settings file that we needed to add periods or there would be an error. I can't remember what throws the error though or why.

**Review by heatherdesigns — APPROVED** [review]
awesome

Everything works as expected! Great job!! I love to take a few minutes for a quick pairing session to learn how you created the `yml` files. And to see if there are other notable things you learned along the way.

I left one comment about syntax. It was more of me thinking out loud. LOL

**Comment by silverli** [comment]
need compiled CSS too!!

File: config/default/field.field.paragraph.hs_row.field_paragraph_style.yml (hunk 1 above)
Comment (silverli) [reply config/default/field.field.paragraph.hs_row.field_paragraph_style.yml:23]: yes! but slightly different enough with styles require to warrant a new class... though maybe I could extend that style in the scss? I'll take a look

File: docroot/modules/humsci/hs_paragraph_types/modules/hs_row/modules/hs_row_display/hs_row_display.module (hunk 2 above)
Comment (silverli) [reply docroot/modules/humsci/hs_paragraph_types/modules/hs_row/modules/hs_row_display/hs_row_display.module]: oh yes. you are right! good catch

File: config/default/field.field.paragraph.hs_row.field_paragraph_style.yml (hunk 1 above)
Comment (silverli) [reply config/default/field.field.paragraph.hs_row.field_paragraph_style.yml:23]: that works!

**Pull request merged**